src/mysql_protocol.erl

```diff
@@ -41,6 +41,7 @@
 %% Macros for pattern matching on packets.
 -define(ok_pattern, <<?OK, _/binary>>).
 -define(error_pattern, <<?ERROR, _/binary>>).
+-define(local_local_data_file, << ?LOAD_LOCAL_FILE_REQ, _/binary >>).
```

Comment:
This is macro for a pattern, so to match the others it should be named "..._pattern" like the others. But, I don't think this macro is needed. See the other comment where this is used.